notification/twilio/twiml_test.go

```diff
@@ -1,19 +1,23 @@
 package twilio
 
 import (
+	"context"
 	"io"
 	"net/http"
 	"net/http/httptest"
 	"testing"
 
 	"github.com/stretchr/testify/assert"
+	"github.com/target/goalert/config"
 )
 
 func TestTwiMLResponse(t *testing.T) {
 	t.Run("hangup", func(t *testing.T) {
+		mockConfig := config.Config{}
```

Comment:
Use `var` for empty declarations, also since it's just a regular config object (not a fake/mock), I think we can call these `cfg`
```suggestion
		var cfg config.Config
```